lib/openstax/aws/build_image_command_1.rb

```diff
@@ -50,6 +50,9 @@ def to_s
         @packer.to_s
       end
 
+      def status 
+        @packer.debug
+      end
```

Comment:
What is this for? `@debug` in packer class doesn't have a reader/accessor

Reply:
yeah this doesn't go here, forgot to remove it